.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 'Feature request'
+about: Making a change to Splink
+title: '<title>'
+assignees: ''
+---
+
+### Is your Pull Request linked to an existing Issue or Pull Request?
+<!--
+  Add links to related issues/prs. For Example "closes #111"
+-->
+
+
+
+### Give a brief description for the solution you have provided
+<!--
+  Provide a clear and concise description of what you want to happen.
+-->
+
+### PR Checklist
+
+- [ ] Add documentation for changes
+- [ ] Add feature to example notebooks at tutorials in [splink_demos](https://github.com/moj-analytical-services/splink_demos) (if appropriate)
+- [ ] Add tests (if appropriate)
+- [ ] Changes are based off the latest version of Splink
```

Comment:
I think these should be in the past tense.

For example, `Added documentation for changes`.